extra_geom/tests/test_pnccd_geometry.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import numpy as np
+from matplotlib.axes import Axes
+
+from extra_geom import PNCCDGeometry
+
+def test_positions():
+    # Positional constants
+    gap_pixels = 50
+    offset_pixels = 10
+
+    # Calculate parameters for the relative/absolute methods
+    top_offset = (-offset_pixels * PNCCDGeometry.pixel_size, 0, 0)
+    bottom_offset = (offset_pixels * PNCCDGeometry.pixel_size, 0, 0)
+    absolute_pos = np.array(
+        [(-PNCCDGeometry.frag_fs_pixels // 2 - offset_pixels,
+          PNCCDGeometry.frag_ss_pixels + gap_pixels // 2,
+          0),
+         (-PNCCDGeometry.frag_fs_pixels // 2 + offset_pixels,
+          -gap_pixels // 2,
+          0)]
+    ) * PNCCDGeometry.pixel_size
+
+    # Create the geometries
+    absolute_geom = PNCCDGeometry.from_absolute_positions(top=absolute_pos[0], bottom=absolute_pos[1])
+    relative_geom = PNCCDGeometry.from_relative_positions(gap=gap_pixels * PNCCDGeometry.pixel_size,
+                                                          top_offset=top_offset,
+                                                          bottom_offset=bottom_offset)
+
+    # They should be the same
+    assert all(np.array_equal(abs_geom_mod[0].centre(), rel_geom_mod[0].centre())
+               for abs_geom_mod, rel_geom_mod
+               in zip(absolute_geom.modules, relative_geom.modules))
```

Comment:
We've got an `assert_geom_close()` utility function which I think could be used here.

("close" there is like numpy and the math module use the word, i.e. essentially equal but with a small tolerance for floating point errors)

Reply:
Ah cool, changed it to use that function (in the squash).